Vpc interconnects

dkeppel626 opened the pull request:
Added Transit Gateway Inventory Script and updated Makefiles and CloudFormation template accordingly.

Commit:
new file:   inventory-transit-gateways.py

Commit:
modified:   inventory-transit-gateways.py

Commit:
modified:   ../Makefile
	modified:   Inventory-Template.yaml
	modified:   ../lambda/Makefile

Commit:
modified:   Makefile
	new file:   inventory-transit-gateway.py

Commit:
deleted:    inventory-transit-gateways.py

jchrisfarris commented on aws-inventory/lambda/inventory-transit-gateway.py:
```diff
@@ -0,0 +1,155 @@
+            resource_item['configuration']                  = tg
+            resource_item['supplementaryConfiguration']     = {}
+            resource_item['resourceId']                     = tg['TransitGatewayId']
+            resource_item['errors']                         = {}
```
Can you add the resource_item['ARN'] mapped to tg['TransitGatewayArn']?

jchrisfarris replied:
You can also map resource_item['resourceCreationTime']        to tg['CreationTime']

Commit:
modified:   ../../aws-inventory/lambda/inventory-transit-gateway.py
	modified:   post-deploy.sh

Commit:
modified:   ../Makefile
	modified:   Inventory-Template.yaml
	modified:   ../lambda/Makefile
	new file:   ../lambda/inventory-client-vpn.py
